Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -226,6 +226,35 @@ void function InitServerBrowserMenu()
 
 	// UI was cut off on some aspect ratios; not perfect
 	UpdateServerInfoBasedOnRes()
+
+        // Listen for map unload to revert forced mod deactivation
+    AddUICallback_OnLevelInit( void function() // There's probably a better spot to put this into
```

Comment:
why not put this in a separate func like most callbacks, just clutters up the already long serverbrowser init func